app/js/directives/CentreDirective.js

```diff
@@ -9,17 +9,16 @@ function CentreDirective() {
     },
     link: function (scope, element, attrs) {
       scope.$watch('attributes.maleAvailability', function (newvalue) {
-        scope.attributes.highlightMale = newvalue < 1 ? 'highlight' : '';
-        scope.attributes.maleAvailability = newvalue === 0 ? 'FULL' : newvalue;
-        scope.attributes.statusMale = newvalue < 1 ? 'No Available Male Beds' : 'Available Male Beds';
-
-      });
```

Comment:
yes third argument is required, apparently, it enforces deeper checking
where are you suggesting to use `!==`?